Casks/l/linqpad@beta.rb

```diff
@@ -0,0 +1,34 @@
+cask "linqpad@beta" do
+  version "8.106.5.5602133"
+  sha256 "6eec41f272f1d6c6e2309d57b93ccc2f2eab14f856fd405db8f7c7789dcb34ea"
+
+  url "https://linqpad.azureedge.net/preview/LINQPad8-Beta.dmg?cache=#{version}",
```

Comment:
```suggestion 
  url "https://linqpad.azureedge.net/preview/LINQPad#{version.major}-Beta.dmg?cache=#{version}",
```

I'm not overly confident that the `cache` parameter will actually return different versions of the software, so we may need to remove the checksum. But I'm ok to test it and see how we go.